client/src/components/TimeTable.jsx

```diff
@@ -0,0 +1,201 @@
+    {
+        super(props);
+        this.state = {
+            courses: [
+                
+                {
+                    Name: "DBMS",
+                    ID: "CS F212",
+                    Days: "Monday Wednesday Friday",
+                    Hours:"5",
+                    Section: "L1"
+                },
+                {
+                    Name: "MicroProcessors",
+                    ID: "CS F241",
+                    Days: "Monday Wednesday Friday",
+                    Hours:"4",
+                    Section: "L1"
+                },
+                {
+                    Name: "Software Engineering",
+                    ID: "IS F341",
+                    Days: "Monday Wednesday Friday",
+                    Hours:"7",
+                    Section: "L1"
+                },
+                {
+                    Name: "EVS",
+                    ID: "BITS F225",
+                    Days: "Tuesday Thursday Saturday",
+                    Hours:"2",
+                    Section: "L1"
+                },
+                {
+                    Name: "Post Colonial Literature",
+                    ID: "HSS F340",
+                    Days: "Tuesday Thursday Saturday",
+                    Hours:"3",
+                    Section: "L1"
+                },
+                {
+                    Name: "MicroProcessors",
+                    ID: "CS F241",
+                    Days: "Tuesday",
+                    Hours:"1",
+                    Section: "T11"
+                },
+                {
+                    Name: "DSA",
+                    ID: "CS F211",
+                    Days: "Thursday",
+                    Hours:"1",
+                    Section: "T3"
+                },
+                
+                {
+                    Name: "DSA",
+                    ID: "CS F211",
+                    Days: "Tuesday Thursday Saturday",
+                    Hours:"5 6 7 8",
+                    Section: "L1"
+                },
+                
+            ]
+        }
+        this.populateTimetable.bind(this);
+        this.gridArray = this.populateTimetable(this.state.courses); 
+    }
+    populateTimetable(coursesAdded)
+    {
+        var gridList = [
+            ["Time",1,2,3,4,5,6,7,8,9,10],
+            ["Monday",0,0,0,0,0,0,0,0,0,0],
+            ["Tuesday",0,0,0,0,0,0,0,0,0,0],
+            ["Wednesday",0,0,0,0,0,0,0,0,0,0],
+            ["Thursday",0,0,0,0,0,0,0,0,0,0],
+            ["Friday",0,0,0,0,0,0,0,0,0,0],
+            ["Saturday",0,0,0,0,0,0,0,0,0,0]
+        ];
+       
+        var Map = {};
+         Map['Monday'] = 1;
+         //Map[1] = 'Monday';
+         Map['Tuesday'] = 2;
+         //Map[2] = 'Tuesday';
+         Map['Wednesday'] = 3;
+         //Map[3] = 'Wednesday';
+         Map['Thursday'] = 4;
+         //Map[4] = 'Thursday';
+         Map['Friday'] = 5;
+         //Map[5] = 'Friday';
+         Map['Saturday'] = 6;
+         //Map[6] = 'Saturday';
```

Comment:
Remove useless commented code

Reply:
We might need the commented code if the logic gets changed due to any reason.

Reply:
That's what version control is for. Make a commit with commented code, and the final one without